tests/api2/test_428_smb_rpc.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+import urllib.parse
+import contextlib
+import pytest
+import sys
+import os
+apifolder = os.getcwd()
+sys.path.append(apifolder)
+from functions import POST, GET, DELETE, SSH_TEST, wait_on_job
+from auto_config import (
+    ip,
+    dev_test,
+    pool_name,
+    password,
+    user,
+)
+from protocols import MS_RPC
+from pytest_dependency import depends
+from utils import create_dataset
+
+reason = 'Skipping for test development testing'
+# comment pytestmark for development testing with --dev-test
+pytestmark = pytest.mark.skipif(dev_test, reason=reason)
+
+SMB_USER = "smbrpcuser"
+SMB_PWD = "smb1234#!@"
+
+
+@contextlib.contextmanager
```

Comment:
You might want to move this to `assets` package so other tests could use this too